src/js/stories/PostCard.stories.js

```diff
@@ -92,6 +92,60 @@ stories.add('Default', () => (
             value: 106,
           },
         ],
+        state: select('Post State', states, 'published'),
```

Comment:
Rather than add a whole other post card you can just use the boolean toggle in the Knobs panel to check how `isNative` behaves when it's true or false.